### ISSUE TITLE
How to show any messages?

### ISSUE THREAD

**Issue by enikao** [ModelingValueGroup/DclareForMPS#3]
The Dclare tool window has several entries for `error`, `warning`, `debug`, etc.

However, I never saw any message there, even though I broke it in different ways:
* Writing conflicting rules
* Getting `ConcurrentModificationException`
* Killing the whole engine (see #2)

How can I enable messages?

**Comment by WimBast** [comment]
Dit you use one of the two last versions? So the release 'Faster and simpler message reporting' (works on 2019.1) or above (works on 2019.2). There is a bug in it before those releases.

**Comment by WimBast** [comment]
Does it work now for you?

**Comment by enikao** [comment]
Yes.

Improvement suggestion: Underline all clickable targets (i.e. the rules and the "..."). I figured out I can see all the rules only by chance.

**Comment by WimBast** [comment]
Good point, thanks. For some reason the underlining of links does not work properly in temporal models in MPS. I will look how to fix this.
